tasking/pulp/tasking/constants.py

```diff
@@ -4,6 +4,20 @@
     # The interval in seconds for which the Celery Process monitor thread sleeps between
     # checking for missing Celery processes.
     CELERY_CHECK_INTERVAL=60,
+    # The interval in seconds during which the Celery Processes being monitored need to report a
+    # hearbeat to be considered active.
+    HEARTBEAT_MAX_AGE=200,
+    # The interval in seconds with which a secondary celerybeat tries to acquire the lock
+    # that is needed to be the primary celerybeat.
+    CELERYBEAT_LOCK_RETRY_TIME=90,
     # Resource manager worker name
```

Comment:
s/Resource manager worker name/The name of the resource manager entry in the workers table/